app/views/course/survey/responses/_response.json.jbuilder

```diff
@@ -5,9 +5,11 @@ json.survey do
 end
 
 json.response do
-  json.(response, :id, :submitted_at)
+  json.(response, :id, :submitted_at, :creator)
   json.sections survey.sections do |section|
     json.(section, :id, :title, :description, :weight)
     json.answers grouped_answers[section.id], partial: 'answer', as: :answer
   end
 end
+
+json.is_creator current_user.id == @survey.creator_id
```

Comment:
Any staff member or coursemology admin should be able to unsubmit for a student. Can do
`json.isStaff = can?(:manage, @survey)` instead.

The abilities are defined in `Course::SurveysAbilityComponent`. Also, currently I'm using snake_case for model attributes since it's cleaner to use `json.(instance, :attr_name)`, and camelCase for derived attributes since it's the JS convention -- save some renaming in the frontend for those.